.vscode/launch.json

```diff
@@ -0,0 +1,36 @@
+// A launch configuration that compiles the extension and then opens it inside a new window
```

Comment:
Thanks for adding these, do we need the configurations, are they helpful?

Reply:
So what it defines is to launch a `npm-watch` task which subsequently looks for  `watch` config (which is `webpack --watch`) in your package.json when debugging task is launched.
What it does is to compile the source code whenever it is changed so developers won't have to restart debugging task everytime they make any changes to code. This file is supposed to be auto-generated when initializing a new extension project using vsce. I personally recommend adding these files to make debugging easier.